README.md

```diff
@@ -27,9 +27,12 @@ Cache Docker Images Whether Built or Pulled
   - [Usage](#usage)
   - [Inputs](#inputs)
     - [Required](#required)
+      - [`key`](#key)
     - [Optional](#optional)
+      - [`read-only`](#read-only)
```

Comment:
This doesn't seem related to this pull request. I'm guessing this change was made our tooling?